Expand the documentation of COVIDcast signals (#103)
cmu-delphi/delphi-epidata: docs/api/covidcast_signals.md

one if they prefer not to answer.

The surveys are just one question long, and ask "Do you know someone in your
community who is sick (fever, along with cough, or shortness of breath, or
difficulty breathing) right now?" Using this survey data, we estimate the
percentage of people in a given location, on a given day, that know somebody who
has a COVID-like illness. Note that this is tracking a different quantity than
the surveys through Facebook, and (unsurprisingly) the estimates here tend to be
much larger.

| Signal | Description |
| --- | --- |
| `raw_cli` | Estimated fraction of people who know someone in their community with COVID-like illness |
| `smoothed_cli` | Estimated fraction of people who know someone in their community with COVID-like illness, smoothed in time |

### `ght`

Data signal based on Google searches, provided to us by Google Health
Trends.  Using this search data, we estimate the volume of COVID-related
searches in a given location, on a given day.  This signal is measured in
arbitrary units (its scale is meaningless); larger numbers represent higher
numbers of COVID-related searches.

| Signal | Description |
| --- | --- |

> | `raw_search` | Google search volume for COVID-related searches, in arbitrary units; normalized by population |

Reviewer: ```suggestion
 | `raw_search` | Google search volume for COVID-related searches, in arbitrary units that are normalized for population |
```

Reviewer: Clarify that arbitrary units are already normalized, rather than having arbitrary units that are then normalized

> | `smoothed_search` | Google search volume for COVID-related searches, in arbitrary units and normalized by population, smoothed in time using a Gaussian linear smoother |

Reviewer: ```suggestion
| `smoothed_search` | Google search volume for COVID-related searches, in arbitrary units that are normalized for population, smoothed in time using a Gaussian linear smoother |
```
Clarify that arbitrary units are already normalized, rather than having arbitrary units that are then normalized

Reviewer: Clarify that arbitrary units are already normalized, rather than having arbitrary units that are then normalized

> | `smoothed_search` | Google search volume for COVID-related searches, in arbitrary units that are normalized for population, smoothed in time using a Gaussian linear smoother |

Reviewer: ```suggestion
 | `smoothed_search` | Google search volume for COVID-related searches, in arbitrary units that are normalized for population, smoothed in time using a local linear smoother with Gaussian kernel |
```
Clarify that smoother is _local_ linear
